lib/src/collections.js

```diff
@@ -129,7 +134,7 @@ const addCollectionsApi = (matroid) => {
   // https://www.matroid.com/docs/api/index.html#api-Collections-PostApiVersionCollectionTasksTaskidScoresQuery
   matroid.queryCollectionByScores = function (taskId, thresholds, numResults) {
```

Comment:
I don't love how numResults is a positional argument here but in configs for query by image. Let's use an configs object here as well.